content/blog/2024-07-07-aws-to-contribute-efficiency-improvements-for-ValKey-8.md

```diff
@@ -0,0 +1,82 @@
++++
+# `title` is how your post will be listed and what will appear at the top of the post
+title= "Unlock 1 Million RPS:  Experience Triple the Speed with Valkey 8.0 "
```

Comment:
I like this name.

Reply:
The title is a bit confusing since Valkey 8.0 isn't being announced with this blog - suggest re-wording.

Reply:
Consider updating the title since this is not an announcement blog of Valkey 8.0.